mk/rt.mk

```diff
@@ -384,7 +384,11 @@ COMPRT_OBJS_$(1) += emutls.o
 endif
 
 ifeq ($$(findstring msvc,$(1)),)
+
+ifeq ($$(findstring freebsd,$(1)),)
```

Comment:
err, this condition is inside one which checks for `msvc`. Pretty sure there’s no combination of freebsd and msvc out there.